deploy/docker/hail_search/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM hailgenetics/hail:0.2.115
+
+LABEL maintainer="Broad TGG"
+
+WORKDIR /hail_search
+
+# Infrastucture Code
+COPY deploy/docker/hail_search/sync_datasets /
```

Comment:
might be a little neater to have a `bin` directory or another scripts directory with `sync_datasets `in it and copy that whole directory, so if we ever need to add another script to our deployment we don't have to keep updating a list here